tensorflow/python/kernel_tests/reader_ops_test.py

```diff
@@ -456,6 +457,45 @@ def testOneEpoch(self):
                                     "\\(requested 1, current size 0\\)"):
         k, v = sess.run([key, value])
 
+  def testZLibFlushRecord(self):
+    fn = os.path.join(self.get_temp_dir(), "tf_record.txt")
+
+    writer = tf.python_io.TFRecordWriter(fn, options=None)
+    writer.write(b"small record")
```

Comment:
if this doesn't work, another module to look into is the tf.compat, which has a lot of functions used to deal with python2 vs. python3 compatibility.  Since this is a test, it's not a big deal, but might make things easier.